test/parallel/test-whatwg-url-origin.js

```diff
@@ -50,3 +51,12 @@ function runURLTests(urltests) {
 
 runURLOriginTests()
 /* eslint-enable */
+
+// Tests below are not from WPT.
+const tests = require(path.join(common.fixturesDir, 'url-tests-additional.js'));
```

Comment:
This doesn't seem to be applicable anymore, now that the tests are moved to `url-tests.js`